collectors/collector.mk

```diff
@@ -19,9 +19,10 @@ compile: deps
 test: compile
 	npm run test
 
-package: test package.zip
+package: clean test package.zip
```

Comment:
please remove `clean`. I still would like to package my local changes in dependencies.

Reply:
np, just a heads up though that `make package` once will succeed, but another `make package` straight after will fail since `package` depends on `test` and we'll have just removed all the dev deps

Reply:
to that matter could you please rework `deps` target to always run `npm install`